spec/features/newspapers_collection_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+RSpec.feature 'Newspapers collection' do
+  let(:collection) { collections(:newspapers) }
+  include_context :collection_with_custom_api_url
+
+  describe 'search' do
+    it 'has an API toggle' do
+      visit collection_path(collection, locale: 'en', q: '')
+
+      expect(page).to have_css('.filter-list a.is-checked .filter-text', text: 'Fulltext')
```

Comment:
Let's use the I18n value here and on the line below already. So in case the text changes it won't break our tests.